recipes/aws-c-event-stream/all/conandata.yml

```diff
@@ -1,19 +1,16 @@
 sources:
+  "0.3.2":
+    url: "https://github.com/awslabs/aws-c-event-stream/archive/v0.3.2.tar.gz"
+    sha256: "3134b35a45e9f9d974c2b78ee44fd2ea0aebc04df80236b80692aa63bee2092e"
   "0.3.1":
     url: "https://github.com/awslabs/aws-c-event-stream/archive/v0.3.1.tar.gz"
     sha256: "bdbc420efc2572689fb167ac288e982a01224876eb79d80e2411fad4c43e9dc0"
+  "0.2.20":
+    url: "https://github.com/awslabs/aws-c-event-stream/archive/v0.2.20.tar.gz"
+    sha256: "a1384c1f63c82a0a0bc64c3e1bc2a672c75614940b71418d96de9e057e31aafd"
   "0.2.15":
     url: "https://github.com/awslabs/aws-c-event-stream/archive/v0.2.15.tar.gz"
     sha256: "4ff2ada07ede3c6afa4b8e6e20de541e717038307f29b38c27efa7c4d875ee26"
-  "0.2.12":
-    url: "https://github.com/awslabs/aws-c-event-stream/archive/v0.2.12.tar.gz"
-    sha256: "cc4ebfe715d8df5b9e3f4a3ce9b67d5f480017a7ebbbfa1d5e64ea53ec672580"
-  "0.2.11":
-    url: "https://github.com/awslabs/aws-c-event-stream/archive/v0.2.11.tar.gz"
-    sha256: "4818b8d3fe02016fcfdd033c1e9d8f6be07ccaeb38664fe8c31c0fd153ea56e6"
-  "0.2.7":
-    url: "https://github.com/awslabs/aws-c-event-stream/archive/v0.2.7.tar.gz"
-    sha256: "bb5c94cdff70c1985fb0b5f30d81756cedb5a3c3075d37f7e1e2b34e2a33c8c0"
```

Comment:
Don't remove this one please, it's still used in aws-crt-cpp and aws-sdk-cpp. I would recommend very high caution when you remove versions of AWS recipes.